sql/sql-server/src/test/scala/org/apache/spark/sql/server/service/postgresql/protocol/v3/PgV3ProtocolScenarioSuite.scala

```diff
@@ -39,7 +39,7 @@ class PgV3ProtocolScenarioSuite extends PgV3ProtocolTest {
 
   testIfSupported("unsupported SQL strings") {
     def expectedErrorMessage(message: String, cmd: String): Seq[String] =
-      Seq(s"Exception detected in '$message'", s"Operation not allowed: $cmd")
+      Seq(s"Exception detected in `$message`", s"Operation not allowed: $cmd")
```

Comment:
Why you changed this?

Reply:
Similar to my other comment, the error message from this exception is different when running on my machine. It may be that there are version differences causing issues. I'm going to close the PR to do a little more research when I get the time. Thanks for taking a look!